YOUR_ADMIN/zca_bootstrap_colors.php

```diff
@@ -16,6 +16,30 @@
 
 $_GET['gID'] = $groupID->fields['configuration_group_id'];
 
+// BOF download SQL file
+$sqlfile = isset($_GET['sqlfile']) ? $_GET['sqlfile'] : '';
+if (zen_not_null($sqlfile)) {
+  $filename = 'zca_bootstrap_colors_' . date('Ymd_His') . '.sql';
+  header('Content-Type: text/plain; charset=utf-8');
+  header('Content-Disposition: attachment; filename=' . $filename);
+  $out = fopen('php://output', 'w');
+
+  $gID = (isset($_GET['gID'])) ? $_GET['gID'] : 1;
```

Comment:
If `$_GET['gID']` isn't set or if `(int)$_GET['gID']` isn't equal to the configuration_group_id associated with the ZCA Colors, the request should be denied.